cardano/src/block/block.rs

```diff
@@ -202,48 +202,46 @@ impl fmt::Display for Block {
     }
 }
 
-impl core::property::Block for Block {
+impl chain_core::property::Block for Block {
```

Comment:
why changing this ?

Reply:
I had to resolve the merge conflict with my own implementation of property::Block for Block, so this is what I picked. Using a consistent name for a crate seems a bit preferable.